platform/AppStoreConnect/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
Just the first file in this directory, but can we move all this under the `ios` directory? Seems irrelevant for Android and others.

Reply:
Well I did put `AppStoreConnect` in `platform` folder with the idea that it could be reused sometime in the future for other Apple platforms, but it could be moved to `ios` folder if we don't think that we will have PolyPod added to HomePod 😄 any time soon.

Reply:
What I meant is to comment on the _AppStoreConnect_ directory which I think should move to _ios_ - was kinda ignoring the `.gitignore`, but I'm with you there, hate the duplication. But maybe a separate PR/commit?